alumet.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+DEFAULT_ALUMET_CONFIG_FILE="/var/lib/alumet/alumet-config.toml"
+
+alumet_bin_name=$(basename "$0")
+path_to_bin=$(temp=$( realpath "$0"  ) && dirname "$temp")
+LIB_ALUMET="${path_to_bin}/../lib/${alumet_bin_name}_bin"
+if [[ -x "$LIB_ALUMET" ]]; then
+    $LIB_ALUMET --config "$DEFAULT_ALUMET_CONFIG_FILE"
```

Comment:
Please pass all the arguments supplied by the user to the agent, with `"$@"` (quotes are important). Related question: what happens if the user wants to override the config? Does it work? Maybe the bash script could avoid using `--config` and use an env variable instead, only if it's not set.